client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -203,6 +205,22 @@ const MarketplaceThankYou = ( { productSlug }: IProps ): JSX.Element => {
 					</FullWidthButton>
 				),
 			},
+			...( documentationURL
+				? [
+						{
+							stepKey: 'whats_next_documentation',
+							stepTitle: translate( 'Documentation' ),
+							stepDescription: translate(
+								'Visit the step-by-step guide to learn how to use this plugin.'
+							),
+							stepCta: (
+								<FullWidthButton href={ documentationURL }>
+									{ translate( 'Visit guide' ) }
+								</FullWidthButton>
+							),
+						},
+				  ]
+				: [] ),
```

Comment:
I would suggest moving this into the second row (than third) cause it provides more value than clicking "Explore Plugins"
![CleanShot 2022-04-27 at 10 56 00@2x](https://user-images.githubusercontent.com/12430020/165469853-1cbade82-eb40-40f8-963c-5ceed3b0e211.png)

Reply:
Fixed at https://github.com/Automattic/wp-calypso/pull/63051/commits/f36e15bdaa3c00f8d279d706c54d2b48b6b65817